### PR TITLE
2.1

### PR DESCRIPTION
**Pull request by patr1ckm**
Releasing 2.1. Manual changes are in first commit.

### CODE REVIEW

**Comment by patr1ckm** [comment]
@crich011 mind reviewing? I'm going to release this as 2.1, and save the new features we proposed for 2.2.

**Comment by crich011** [comment]
Will do! I'll look it over now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Changelog
 
-## Unreleased
+## [2.1.0] - 2019-09-06
 
 ### Changed
 - `run_template` takes an additional value of JSONValue. If TRUE,
-  function returns the JSON output instead of the fileid.
+  function returns the JSON output instead of the file id.
 - All API calls now retry a max of 10 times for maximum of 1 hr when rate limited. (#214).
```
Comment (crich011) [inline CHANGELOG.md]: Naive question, but is the additional diff not reflected in the changelog because they appear to just be small changes to docs?

Comment (patr1ckm) [reply CHANGELOG.md]: Yes, 99% of this PR is autogenerated. The only changes I made by hand are in the first commit.

**Comment by patr1ckm** [comment]
@crich011 can you hit the 'approve' button? Thank you!

**Pull request merged**